image/store.go

```diff
@@ -213,6 +215,46 @@ func (is *store) Get(id ID) (*Image, error) {
 	return img, nil
 }
 
+func (is *store) GetTarSeekStream(id ID) (ioutils.ReadSeekCloser, error) {
```

Comment:
Please consider to add a documentation to this function. something like
// GetTarSeekStream concatenates streams of Tars